Makefile

```
@@ -81,14 +96,17 @@ clean-db:
 	podman rm -f quipucords-dev-db || true
 	podman volume rm -f quipucords-dev-db
 
+sync:
+	$(LIBSSH_BUILD_ENV) uv sync
+
 lock-requirements: lock-main-requirements lock-build-requirements
```

checkmake (0.3.2)

[warning] Target "lock-requirements" should be declared PHONY.

(phonydeclared)